extensions/amp-video/1.0/storybook/Basic.amp.js

```diff
@@ -1,48 +1,29 @@
 import {withAmp} from '@ampproject/storybook-addon';
-import {boolean, number, object, text, withKnobs} from '@storybook/addon-knobs';
 
 import * as Preact from '#preact';
 
 import {VideoElementWithActions} from './_helpers';
 
 export default {
   title: 'amp-video-1_0',
-  decorators: [withKnobs, withAmp],
+  decorators: [withAmp],
   parameters: {
     extensions: [
       {name: 'amp-video', version: '1.0'},
       {name: 'amp-accordion', version: '1.0'},
     ],
     experiments: ['bento'],
   },
-};
-
-const AmpVideoWithKnobs = ({i, ...rest}) => {
-  const group = i ? `Player ${i + 1}` : undefined;
-
-  const width = text('width', '640px', group);
```

Comment:
question - do you know if storybook controls supports something like knob grouping? all these knobs use a group, so not sure if there's an analogue for controls

Reply:
Right now grouping isnt supported under controls, it looks like there are folks who have kept knobs around specifically for this feature. So we are losing this functionality in the migration.